### PR TITLE
Added support for raw unicode literals

### LINKED ISSUE #47: Add support for unicode literals?
First of all, thanks for this amazing middleware!

While I was using, there were situations to log Korean request and/or response entities.  
However, I realized that this middleware logs unicode literals "as is", not decoded into its natural characters. (i.e. Korean, Chinese, etc.)

Wouldn't it be nice if I can add supports for those unicode literals?

### PR DESCRIPTION
**Pull request by gyukebox**
Resolves #47

### CODE REVIEW

File: tests.py [hunk 1]
```diff
@@ -11,8 +11,9 @@
 from django.http import HttpResponse
 
 import request_logging
-from request_logging.middleware import LoggingMiddleware, DEFAULT_LOG_LEVEL, DEFAULT_COLORIZE, DEFAULT_MAX_BODY_LENGTH,\
-    NO_LOGGING_MSG
+from request_logging.middleware import \
```
Comment (tclancy) [inline tests.py]: Can you add a quick test to prove the expected behavior with Unicode content? Looks like there are also some conflicts to merge.

Comment (gyukebox) [reply tests.py]: Added them in 86d35c, but whoops :(
I'll come back after figuring out what's wrong

**Comment by tclancy** [comment]
Can you resolve the issues here and I will make another release? Thanks.

**Comment by gyukebox** [comment]
@tclancy Would you mind adding `wontfix` label to #47 and close this for now?  
Sorry for not providing further updates upon this issue due to lack of understanding to Python 2.7 and lack of time 😭

**Comment by tclancy** [comment]
No worries, I'm ok with leaving this open in case someone else wants to pick it up (or I get time). If nothing else, it should help prevent dupes and provide a search result for people having problems with the issue.
